README.md

```diff
@@ -10,14 +10,15 @@ The Node.js End User Feedback team works with Node.js end users to provide a bi-
 * Support [CommComm](https://github.com/nodejs/community-committee) and [TSC](https://github.com/nodejs/community-committee) end user feedback and outreach needs.
 
 ### Governance and Contributing
-The Node.js End User Feedback team has adopted the core governance and contributing policies of Node.js.
+The Node.js End User Feedback team has adopted the core governance and contribution policies of the Node.js project.
```

Comment:
> Node.js End User Feedback team

I believe this is an initiative and not a team - team is a term that's used in the project under the TSC. Not sure if that term is still used, but this should probably reflect that this is an initiative.